CityOfDetroit/bloom
# Feature/add eligibility disclaimer

## Issue

Add an eligibility disclaimer to the questionnaire.

- Addresses #691

## Description

This PR adds a disclaimer as a final page in the eligibility questionnaire before proceeding to viewing the filtered listings. I'm not closing #691 just yet, since I wasn't sure how to handle the AMI calculation page with this change. I took the liberty of changing the "View Listing" text on all the steps to say "Finish" instead, which brings the user to the disclaimer, since "View Listings" wouldn't make sense if it was bringing the user to the disclaimer.

Desktop:
![localhost_3000_eligibility_household](https://user-images.githubusercontent.com/66751489/141348405-89a5cc58-8636-4955-b5d2-8835687e3f7b.png)

Mobile:
![localhost_3000_eligibility_household (1)](https://user-images.githubusercontent.com/66751489/141348433-5935b1bb-f0f9-4880-a232-49ca580b738f.png)

## Type of change


## How Can This Be Tested/Reviewed?

Click "Do I Qualify?" on the home page, and go through the questionnaire. Clicking "Finish" at any point should bring you to the disclaimer, and then clicking "View Listings" should bring you to the filtered listings view.

Unit tests: `cd sites/public && yarn test:unit-tests`
Cypress tests: `cd sites/public && yarn test:headless --spec 'cypress/integration/eligibility-questionnaire.ts'`

## Checklist:
